crates/common_enums/src/enums.rs

```diff
@@ -37,3 +38,2097 @@ pub enum CountryCode {
+    ElSalvador,
+    EquatorialGuinea,
+    Eritrea,
+    Estonia,
+    Ethiopia,
+    FalklandIslandsMalvinas,
+    FaroeIslands,
+    Fiji,
+    Finland,
+    France,
+    FrenchGuiana,
+    FrenchPolynesia,
+    FrenchSouthernTerritories,
+    Gabon,
+    Gambia,
+    Georgia,
+    Germany,
+    Ghana,
+    Gibraltar,
+    Greece,
+    Greenland,
+    Grenada,
+    Guadeloupe,
+    Guam,
+    Guatemala,
+    Guernsey,
+    Guinea,
+    GuineaBissau,
+    Guyana,
+    Haiti,
+    HeardIslandandMcDonaldIslands,
+    HolySee,
+    Honduras,
+    HongKong,
+    Hungary,
+    Iceland,
+    India,
+    Indonesia,
+    IranIslamicRepublicof,
+    Iraq,
+    Ireland,
+    IsleofMan,
+    Israel,
+    Italy,
+    Jamaica,
+    Japan,
+    Jersey,
+    Jordan,
+    Kazakhstan,
+    Kenya,
+    Kiribati,
+    KoreaDemocraticPeoplesRepublicof,
+    KoreaRepublicof,
+    Kuwait,
+    Kyrgyzstan,
+    LaoPeoplesDemocraticRepublic,
+    Latvia,
+    Lebanon,
+    Lesotho,
+    Liberia,
+    Libya,
+    Liechtenstein,
+    Lithuania,
+    Luxembourg,
+    Macao,
+    MacedoniatheformerYugoslavRepublicof,
+    Madagascar,
+    Malawi,
+    Malaysia,
+    Maldives,
+    Mali,
+    Malta,
+    MarshallIslands,
+    Martinique,
+    Mauritania,
+    Mauritius,
+    Mayotte,
+    Mexico,
+    MicronesiaFederatedStatesof,
+    MoldovaRepublicof,
+    Monaco,
+    Mongolia,
+    Montenegro,
+    Montserrat,
+    Morocco,
+    Mozambique,
+    Myanmar,
+    Namibia,
+    Nauru,
+    Nepal,
+    Netherlands,
+    NewCaledonia,
+    NewZealand,
+    Nicaragua,
+    Niger,
+    Nigeria,
+    Niue,
+    NorfolkIsland,
+    NorthernMarianaIslands,
+    Norway,
+    Oman,
+    Pakistan,
+    Palau,
+    PalestineStateof,
+    Panama,
+    PapuaNewGuinea,
+    Paraguay,
+    Peru,
+    Philippines,
+    Pitcairn,
+    Poland,
+    Portugal,
+    PuertoRico,
+    Qatar,
+    Réunion,
+    Romania,
+    RussianFederation,
+    Rwanda,
+    SaintBarthélemy,
+    SaintHelenaAscensionandTristandaCunha,
+    SaintKittsandNevis,
+    SaintLucia,
+    SaintMartinFrenchpart,
+    SaintPierreandMiquelon,
+    SaintVincentandtheGrenadines,
+    Samoa,
+    SanMarino,
+    SaoTomeandPrincipe,
+    SaudiArabia,
+    Senegal,
+    Serbia,
+    Seychelles,
+    SierraLeone,
+    Singapore,
+    SintMaartenDutchpart,
+    Slovakia,
+    Slovenia,
+    SolomonIslands,
+    Somalia,
+    SouthAfrica,
+    SouthGeorgiaandtheSouthSandwichIslands,
+    SouthSudan,
+    Spain,
+    SriLanka,
+    Sudan,
+    Suriname,
+    SvalbardandJanMayen,
+    Swaziland,
+    Sweden,
+    Switzerland,
+    SyrianArabRepublic,
+    TaiwanProvinceofChina,
+    Tajikistan,
+    TanzaniaUnitedRepublicof,
+    Thailand,
+    TimorLeste,
+    Togo,
+    Tokelau,
+    Tonga,
+    TrinidadandTobago,
+    Tunisia,
+    Turkey,
+    Turkmenistan,
+    TurksandCaicosIslands,
+    Tuvalu,
+    Uganda,
+    Ukraine,
+    UnitedArabEmirates,
+    UnitedKingdomofGreatBritainandNorthernIreland,
+    UnitedStatesofAmerica,
```

Comment:
```suggestion
    UnitedStatesOfAmerica,
```

We'd prefer if the names of countries followed `PascalCase` (or `UpperCamelCase`) for all words. Please follow this for words like "of" and "and" (and anything I may have left out) as well, so these would be `Of` and `And` respectively. Please update other country names to follow this naming convention as well.